website/files/models/osfstorage.py

```diff
@@ -81,10 +84,42 @@ def move_under(self, destination_parent, name=None):
             raise exceptions.FileNodeCheckedOutError()
         return super(OsfStorageFileNode, self).move_under(destination_parent, name)
 
-    def save(self):
+    def save(self):  # TODO write log
+        """
+        Uses OsfSFN save to save the node,
```

Comment:
Docstring is wordy and uses a non-immediately-apparent abbreviation. Consider something like 
>Saves node, logs any check-out or check-in actions